File: JavaScript_Courses/JavaScript_Basics/JS_BP_2022_10/02_Conditional_Statements_Exercise/01_Sum_Seconds.js
function sumSeconds(input) {  // ["35", "45", "44"]
    let secondsFirst = Number(input[0]);  // 35
    let secondsSecond = Number(input[1]);
    let secondsThird = Number(input[2]);

    let secondsSum = secondsFirst + secondsSecond + secondsThird;

    let minutes = Math.floor(secondsSum / 60);
    let seconds = secondsSum % 60;

    // console.log(`${minutes}:${seconds}`);
    
    if (seconds < 10) {
    console.log(`${minutes}:0${seconds}`);
    } else {
    console.log(`${minutes}:${seconds}`);
    }
}

// sumSeconds(["35", "45", "44"]);

File: Python_Courses/Python_Basics/Python_PB_2023_10/02_Conditional_Statements_Exercise/01_Sum_Seconds.py
seconds_first = int(input())
seconds_second = int(input())
seconds_third = int(input())

total_seconds = seconds_first + seconds_second + seconds_third

minutes = total_seconds // 60  # 125 => 2           125 // 60 => 2   120 // 60 => 2
seconds = total_seconds % 60   # 125 => 5

print(f"{minutes}:{seconds :02d}")
